index.html

```diff
@@ -349,6 +349,28 @@ <h2>About Myself</h2>
 
 
 
+      </section>
+      <section class="contact-form">
+        <h2>Contact Me</h2>
+        <p>If you have an application you are interested in developing, a
+          feature that you need built or a project that needs coding.
+          I’d love to help with it!</p>
+        <form action="https://formspree.io/f/xoqzgvow" method="post"
+          enctype="multipart/form-data">
+          <input type="text" placeholder="Full names"
+            maxlength="30" value="Lorem ipsum" minlength="10"
```

Comment:
- [ ] As suggested by the previous reviewer please kindly remove the minLength attribute from here as it is not a part of the project requirements, and it is very possible for a person's full name length to be less than 10 charaters e.g "John Doe" is 8 characters.

- [ ] Kindly consider putting the Lorem ipsum into the placeholder attribute, as it is not a good idea to be filling in the Full name inputs for the user.